addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -370,3 +370,27 @@ def test_mailing_shortener(self):
                     link_info,
                     link_params=link_params,
                 )
+
+    class TestMailingScheduleDateWizard(MassMailCommon):
```

Comment:
Strange indentation, seems there is one unnecessary tab indent hanging

Reply:
Corrected.